run_service.sh

```diff
@@ -531,20 +524,40 @@ try_read_storage() {
         fi
 
         # INFO: This is a fix to avoid corrupting already-created stores
-        if [ "$USE_STAKING" = true ]; then
-            # Existing staking services use AGENT_ID=12
-            AGENT_ID=12
-        else
-            # Existing non-staking services use AGENT_ID=14
+        # We set by default AGENT_ID=14, and check if we are in the Everest
+        # staking program before on-chain actions to correct AGENT_ID=12 if 
+        # necessary.
```

Comment:
I think this comment would make more sense here:

https://github.com/valory-xyz/trader-quickstart/blob/dc672fefa6b2ebc884a15b9438dd4279ce0eb60c/run_service.sh#L758-L759

Reply:
Ok